sympy/physics/optics/utils.py

```diff
@@ -444,9 +444,9 @@ def critical_angle(medium1, medium2):
 
 
 
-def lens_makers_formula(n_lens, n_surr, r1, r2):
+def lens_makers_formula(n_lens, n_surr, r1, r2=None, *, d=None, lens='thin'):
```

Comment:
This signature is a bit weird. What about just having separate functions for the three lens types?

Reply:
Having separate function for such a simple thing doesn't seem necessary. I'll try to improve this API.

Reply:
So let's make the signature `lens_makers_formula(n_lens, n_surr, r1, r2, d=0)`?

Reply:
Yes, I'll do that soon. Just a bit busy to push changes.